src/ensembl/src/content/app/browser/browserActions.ts

```diff
@@ -270,18 +249,14 @@ export const updateApplyToAll = createAction(
   'browser/update-apply-to-all',
   (resolve) => {
     return (yn: boolean) => {
```

Comment:
If `yn` means yes/no, as I suspect it does, can we change this argument's name to `yesNo`?

Reply:
Changed this createStandardAction.